mmedit/models/losses/feature_loss.py

```diff
@@ -0,0 +1,94 @@
+import torch
+import torch.nn as nn
+from mmcv.runner import load_checkpoint
+
+from mmedit.models.components.discriminators import LightCNN
+from mmedit.utils import get_root_logger
+from ..registry import LOSSES
+
+
+class LightCNNFeature(nn.Module):
+    """Feature of LightCNN.
+
+    It is used to train DICGAN.
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        model = LightCNN(3)
+        self.features = nn.Sequential(*list(model.features.children()))
+        for _, v in self.features.named_parameters():
+            v.requires_grad = False
+
+    def forward(self, x):
+        """Forward function.
+
+        Args:
+            x (Tensor): Input tensor.
+
+        Returns:
+            Tensor: Forward results.
+        """
+
+        return self.features(x)
+
+    def init_weights(self, pretrained=None, strict=True):
+        """Init weights for models.
+
+        Args:
+            pretrained (str, optional): Path for pretrained weights. If given
+                None, pretrained weights will not be loaded. Defaults to None.
+            strict (boo, optional): Whether strictly load the pretrained model.
+                Defaults to True.
+        """
+        if isinstance(pretrained, str):
+            logger = get_root_logger()
+            load_checkpoint(self, pretrained, strict=strict, logger=logger)
+        elif pretrained is not None:
+            raise TypeError(f'"pretrained" must be a str or None. '
+                            f'But received {type(pretrained)}.')
+
+
+@LOSSES.register_module()
+class FeatureLoss(nn.Module):
```

Comment:
I think this naming is a bit too general. Since it is specfic to the LightCNN, maybe change to a more specific name?

Or maybe this loss can be moved to model, by computing the feature and directly use L1Loss?